src/test/java/com/microsoft/azure/relay/TestUtil.java

```diff
@@ -9,4 +10,23 @@
 	public static final String ENTITY_PATH = CONNECTION_STRING_BUILDER.getEntityPath();
 	public static final String KEY_NAME = CONNECTION_STRING_BUILDER.getSharedAccessKeyName();
 	public static final String KEY = CONNECTION_STRING_BUILDER.getSharedAccessKey();
+		
+	@SuppressWarnings("unchecked")
+	public static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable) {
```

Comment:
>assertThrows [](start = 39, length = 12)

Do you mind adding a bit of documentation to the function?

 /**
  * Asserts that the execution of the given Executable will throw the expected exception
  * @param expectedType The class of the expected exception
  * @param executable The code to be executed
  * @return The actual exception that's thrown, if it matches the expected exception type
  */

Reply:
Function moved to Assertions.java and documentation added there.

---
In reply to: [259062359](https://github.com/Azure/azure-relay-java/pull/20#discussion_r259062359) [](ancestors = 259062359)